src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,86 @@
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
+  const linkNumbers = getNumbers(1, linksAmount);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
The `href` attribute is set to `#prev`, which might cause the page to scroll to the top when clicked. Consider using a button instead of an anchor tag if the link is not navigating to a different URL.